app/herdbook.py

```diff
@@ -770,12 +771,12 @@ def issue_certificate(i_number):
         form = request.json
 
         ind_data.update(**form, issue_digital=True)
+        # keep the ind_data object intact
+        ind_data_copy = copy.copy(ind_data)
         res = da.update_individual(ind_data, session.get("user_id", None))
```

Comment:
Is `update_individual` equipped to handle these dicts with Individuals in them?

Reply:
Nothing has changed in that regard. I think it still expects a form, and that's what it currently gets.

Reply:
Ok, I guess nothing has changed at least (if I read correctly, what it gets is one of those dicts with Individuals in them).